csharp/source/autowrap/csharp/csharp.d

```diff
@@ -1,5 +1,7 @@
 module autowrap.csharp.csharp;
 
+import scriptlike;
```

Comment:
At most, `import scriptlike: interp;`.

Reply:
It's a mixin that dissolves into other publicly imported symbols. I can keep adding them tell I get it to work, but you're going to have symbols in the import list that are never used in the code.